index.bs

```diff
@@ -44,6 +44,11 @@ spec: sensors; urlPrefix: https://w3c.github.io/sensors/#
 spec: manifest; urlPrefix: https://w3c.github.io/manifest/#
     type: dfn
         text: install; url: dfn-install
+spec: feature-policy; urlPrefix: https://wicg.github.io/feature-policy/#
+    type: dfn
+        text: policy-controlled feature
+    type: dfn
+        text: feature name
```

Comment:
This reference doesn't exist in the Feature Policy spec anymore. Maybe use wording like in https://wicg.github.io/feature-policy/#parse-policy-directive:

> Let _feature_ be the [=policy-controlled feature=] identified by _feature-name_.